src/main/java/org/mockito/internal/util/MockUtil.java

```diff
@@ -55,21 +56,25 @@ public static TypeMockability typeMockabilityOf(Class<?> type) {
         return mock;
     }
 
-    public static <T> void resetMock(T mock) {
+    public static void resetMock(Object mock) {
```

Comment:
Why were the generic parameters removed from this and the other methods? I understand that they are not used, but I wonder if there are unintended side-effects of this change.

Reply:
Just as a cleanup on the side since I am touching the code already. The generic type is erased to `Object` and has no other function, that's why I removed it. No impact on source or binary compatibility.